### PR TITLE
fix(css): Make auth modal less wonky

### PR DESCRIPTION
**Pull request by ceorourke**
**Before**
Screen Shot 2021-07-22 at 10 25 18 AM

**After**
Screen Shot 2021-07-22 at 10 25 44 AM

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

File: src/sentry/templates/sentry/integrations/slack-link-team.html [hunk 1]
```diff
@@ -18,15 +18,15 @@
     <p>Choose which team you'd like to link to the <b>#{{ channel_name }}</b> channel.</p>
 
     <div class="align-center" style="padding-bottom:20px;">
-      <select id="{{ form.team.id_for_label }}" name="{{ form.team.html_name }}" class="form-control">
+      <select id="{{ form.team.id_for_label }}" data-test-id="select-team" name="{{ form.team.html_name }}" class="form-control">
```
Comment (billyvg) [inline src/sentry/templates/sentry/integrations/slack-link-team.html:21]: You could also query by the `name` attribute e.g. `[name="<whatever form.team.html_name is>"]`

File: src/sentry/templates/sentry/integrations/slack-link-team.html [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
         {% for team in teams %}
-        <option value="{{ team.id }}">{{ team.slug }}</option>
+        <option data-test-id="my-team" value="{{ team.id }}">{{ team.slug }}</option>
```
Comment (billyvg) [inline src/sentry/templates/sentry/integrations/slack-link-team.html:23]: Similarly here `[value="<teamId>"]`

File: tests/acceptance/test_link_team.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    ExternalProviders,
+    Identity,
+    IdentityProvider,
+    IdentityStatus,
+    Integration,
+)
+from sentry.testutils import AcceptanceTestCase
+
+
+class AcceptOrganizationInviteTest(AcceptanceTestCase):
```
Comment (billyvg) [inline tests/acceptance/test_link_team.py:15]: Should this be named something else?

File: tests/acceptance/test_link_team.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        self.path = linking_url.path
+
+    def test_link_team(self):
+        with self.feature("organizations:notification-platform"):
+            self.login_as(self.user)
+            self.browser.get(self.path)
+            self.browser.wait_until_not(".loading")
+            self.browser.click('[data-test-id="select-team"]')
+            self.browser.click('[data-test-id="my-team"]')
+            self.browser.snapshot(name="select team")
```
Comment (billyvg) [inline tests/acceptance/test_link_team.py:65]: I would probably prefix this with `link team -` or `slack link team -` or something so as to not conflict with snapshot name as `select team` is a bit generic.

**Review by billyvg — COMMENTED** [review]
👍 👍 👍

File: tests/acceptance/test_link_team.py (hunk 3 above)
Comment (ceorourke) [reply tests/acceptance/test_link_team.py:15]: lol oops bad copy paste

**Review by billyvg — APPROVED** [review]

**Pull request merged**